variables.tf

```diff
@@ -0,0 +1,80 @@
+/*
+Module variables
+*/
+
+variable "alb_is_internal" {
+  description = "Determines if the ALB is internal. Default: false"
+  default     = false
+}
+
+variable "alb_name" {
+  description = "The name of the ALB as will show in the AWS EC2 ELB console."
+  default     = "my-alb"
+}
+
+variable "alb_protocols" {
+  description = "A comma delimited list of the protocols the ALB accepts. e.g.: HTTPS"
+  default     = "HTTPS"
+}
+
+variable "alb_security_groups" {
+  description = "A comma separated string of security groups with which we associate the ALB. e.g. 'sg-edcd9784,sg-edcd9785'"
+  type        = "list"
+}
+
+variable "aws_region" {
```

Comment:
Should not be possible to customize in the module itself.

Reply:
Unsure what's meant by this. Can you explain?